declarative-schedule-doc.md

```diff
@@ -196,6 +197,39 @@ test_data:
         - size: 3mb
   !include: path/to/test_data.yaml
 ```
+Test data sometimes are more related to a particular schedule, sometimes to other test data shared with other test suites and sometimes it is a mix. In those cases, `YAML_DATA` setting can be used to give us the flexibility to avoid duplicate schedule files just because they have different data and due to it will be pointing to a test data file and at the moment recursive inclusion is not implemented (to reduce complexity),only for this particular case, is allowed the possibility to use `!include` functionality in test data file, not cutting any path for the tester. For instance:
+
+One job could be running with:
+```
+YAML_DATA=path/to/test_data_for_one_test_suite.yaml
+```
+And another job with:
+```
+YAML_DATA=path/to/another_test_data_for_the_same_test_suite.yaml
```

Comment:
Let's include an example for yaml format of the job group, to show how to override it. If you don't mind.

Reply:
I was not sure as not everybody is using job group yaml, but it makes sense.